pkg/syncer/fullsync.go

```diff
@@ -73,6 +73,11 @@ func csiFullSync(ctx context.Context, metadataSyncer *metadataSyncInformer) {
 	log.Debugf("FullSync: pvToPVCMap %v", pvToPVCMap)
 	log.Debugf("FullSync: pvcToPodMap %v", pvcToPodMap)
 
+	// inlineVolumeMap holds the volume path information for migrated volumes which are used by Pods
+	inlineVolumeMap, err := getInlineMigratedVolumesInfo(ctx, metadataSyncer)
```

Comment:
Since `inlineVolumeMap` is only used inside `getVolumesToBeDeleted`. Can you move retrival of inlineVolumeMap in the `getVolumesToBeDeleted` instread of passing as an argument?